Src/Extern/SUNDIALS/AMReX_SUNMemory.cpp

```diff
@@ -38,11 +38,11 @@ namespace {
   {
     SUNMemory mem = SUNMemoryNewEmpty();
 
+    if (mem == nullptr) return 0;
```

Comment:
Just noticed this should be `return -1`
```suggestion
    if (mem == nullptr) return -1;
```